Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -150,6 +150,32 @@ function rotateData(obj,axis,value)
             obj.rotateTable(obj.forces, axis, value);
             disp('Marker and Force tables have been rotated')
         end
+        function convertMillimeters2Meters(obj)
+            % Function to convert displacement forceplate measurements made
+            % in millimeters to meters. This will convert point data (mm)
+            % to m and Torque data (Nmm) to Nm.
+            import org.opensim.modeling.*
+            
+            nRows  = obj.forces.getNumRows();
+            labels = obj.forces.getColumnLabels();
+            
+            for i = 0 : obj.forces.getNumColumns - 1
+                % All force columns will have the 'f' prefix while point
+                % and moment columns will have 'p' and 'm' prefixes,
+                % respectively. 
+                if ~contains(char(labels.get(i)),'f')
```

Comment:
Maybe unrelated to this PR but is the 'f' prefix an arbitrary convention we made or is it enforced by C3D?

Reply:
Yep, forces are labelled _fn_, where n is the force number. We use _p_ for COP and _m_ for Moment